### PR TITLE
Aaa base for testsuites machines

### PR DESCRIPTION
**Pull request by MalloZup**

### CODE REVIEW

**Comment by MalloZup** [comment]
https://github.com/moio/sumaform/issues/43

File: salt/client/testsuite.sls [hunk 1]
```diff
@@ -20,6 +20,7 @@ cucumber-requisites:
       - man
       - wget
       - adaptec-firmware
+      - aaa_base-extras
```
Comment (moio) [inline salt/client/testsuite.sls]: This does not exist in CentOS as far as I know. We have to guard against non-SUSE OS's now that we support it.

Comment (MalloZup) [inline salt/client/testsuite.sls]: Ok,  doing jonja ninja then

**Comment by MalloZup** [comment]
@moio, ok for me

**Pull request merged**